gapis/api/vulkan/api/renderpass_framebuffer.api

```diff
@@ -405,6 +412,29 @@ sub ref!RenderPassObject createRenderPassObjectFromInfo2(
       ViewMask: subpass.viewMask,
       PipelineBindPoint: subpass.pipelineBindPoint,
     )
+    if subpass.pNext != null {
+      numPNext := numberOfPNext(subpass.pNext)
+      next := MutableVoidPtr(as!void*(subpass.pNext))
+      for j in (0 .. numPNext) {
+        sType := as!const VkStructureType*(next.Ptr)[0]
+        switch sType {
+          case VK_STRUCTURE_TYPE_SUBPASS_DESCRIPTION_DEPTH_STENCIL_RESOLVE: {
+            ext := as!VkSubpassDescriptionDepthStencilResolve*(next.Ptr)[0]
+            description.DepthStencilResolve = new!SubpassDescriptionDepthStencilResolve(
+              DepthResolveMode: ext.depthResolveMode,
+              StencilResolveMode: ext.stencilResolveMode,
+            )
+            attachment := ext.pDepthStencilResolveAttachment[0]
+            description.DepthStencilResolve.DepthStencilResolveAttachment = AttachmentReference(
```

Comment:
We are missing pNext handling here.

Reply:
@yalcinmelihyasin, just to be clear, the other `AttachmentReference()` usages on lines 442, 453, 465 and 476 also need 'pNext' handling, yes?

Reply:
Yes. with the new PR merging, we are able to do this now.